arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -173,6 +173,125 @@
         }
       }
     },
+    "/{scope}/providers/Microsoft.Authorization/locks/{lockName}": {
+      "put": {
+        "tags": [
+          "ManagementLocks"
+        ],
+        "operationId": "ManagementLocks_CreateOrUpdateByScope",
+        "description": "Create or update a management lock by scope.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope for the lock. "
```

Comment:
@mcardosos - ROFL . I made the same comment as you have fixed. Your fix and my comment were 4 seconds apart.